docs/administration/vulnerability-scanning/_index.md

```diff
@@ -3,13 +3,27 @@ title: Vulnerability Scanning
 weight: 35
 ---
 
-Harbor provides static analysis of vulnerabilities in images through the open source [Clair](https://github.com/coreos/clair) project. 
+Harbor provides static analysis of vulnerabilities in images through the open source projects [Trivy](https://github.com/aquasecurity/trivy) and [Clair](https://github.com/coreos/clair). To be able to use Trivy, Clair or both you must have enabled Trivy, Clair or both when you installed your Harbor instance (by appending installation options `--with-trivy`, `--with-clair` or both).
 
 {{< important >}}
-Clair is an optional component. To be able to use Clair you must have enabled Clair when you installed your Harbor instance.
+So far, Harbor supports only one system default scanner. The following principles will be applied to determine the system default scanner among the default installed scanners.
```

Comment:
'So far' -> 'Currently, '

Reply:
'The following principles will be applied to determine the system default scanner among the default installed scanners.  - >  'The following logic determines the behavior of the default scanner for new installation as well as for upgrades'